qa/L0_io/test.sh

```diff
@@ -67,9 +67,7 @@ for trial in graphdef savedmodel onnx libtorch plan python python_dlpack; do
             cp ../python_models/add_sub/model.py $MODELSDIR/${full}/1/. && \
             cp ../python_models/add_sub/config.pbtxt $MODELSDIR/${full}/. && \
             (cd $MODELSDIR/${full} && \
-                    sed -i "s/label_filename:.*//" config.pbtxt && \
-                    sed -i "0,/name:.*/{s/name:.*/name: \"${full}\"/}" config.pbtxt && \
-                                        echo "max_batch_size: 64" >> config.pbtxt)
```

Comment:
I think the max_batch_size part should stay in, but removing the "name" `sed` LGTM

Reply:
Good catch